lib/logstash/filters/elasticsearch.rb

```diff
@@ -174,6 +173,9 @@ def register
 
     test_connection!
     setup_serverless
+    if get_client.es_transport_client_type == "elasticsearch_transport"
+      require_relative "elasticsearch/patches/_elasticsearch_transport_http_manticore"
```

Comment:
review note: `_elasticsearch_transport_http_manticore` sets user-agent and accept-encoding headers for the LS versions between 7.2 and 7.16. It seems to me we can remove this but I have kept so far as there are active 7.x users.